### DISCUSSION TITLE
Free version + free icons not showing with local library

### DISCUSSION THREAD

**Discussion by unluckychild** [FortAwesome/Font-Awesome#19507] [Q&A] [answered]
I have blank page with code below. If the library is from CDN, everything works fine. If i download that SAME library and use it locally, SOME icons are not showing. (deleted cache, ctrl f5 refresh, checked link)

I am using free version + free icons + version 6.21 + icons are in that version.

code below

this works
`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
Circle: <i class="fa-solid fa-arrows-to-circle"></i><br />
Download: <i class="fa-solid fa-download"></i>`

this one of them works, eventhough both are free and in in same version:
`<link rel="stylesheet" href="css/all.min.css">
<i class="fa-solid fa-arrows-to-circle"></i> --- doesnt work
<i class="fa-solid fa-download"></i> -- works
`

edit SOLVED, forgot about webfonts folder.

**Comment by NeoStarkX** [comment]
I think it is better to post your answer and select it as the solution :)

**Comment by tagliala** [comment, marked as answer]
In case of self-hosting the webfont/css implementation, both `/webfonts` and `/css` folders should be updated

Please also make sure that there are no cache issues. Fingerprinting assets is a good solution

Ref:
- https://fontawesome.com/v5/docs/web/setup/host-font-awesome-yourself#using-web-fonts-with-css
